Chart.yaml

```diff
@@ -3,4 +3,4 @@ apiVersion: v2
 name: task-containers
 description: Contains the Tekton Tasks to manage container images
 type: application
-version: 0.3.3
+version: 0.4.0
```

Comment:
why these changes comes if you rebase :thinking: 
as these changes are already merged